Report/sections/1_analysis/Importance_of_programming.tex

```diff
@@ -0,0 +1,56 @@
+\chapter{Importance of Programming} \label{chap:importance_of_programming}
+
+Humans have entered a digital age, as more and more stuff gets 
+digitalized every year. This transition has been ongoing for a few 
+generations, and the need for software cannot be understated. The 
+world runs on software, your coffee machine, television, car, mobile
+phone, and the list goes on, not to mention that every single business 
+and organization in the world needs an app, website, etc. Most companies
+need custom made applications or websites, and therefore a lot of coders
+is needed.
+
+This digital transition has made humans more dependant on technology than ever - 
+and for a good reason. The year 2020 when the pandemic hit the world is a good example of that.
+The pandemic changed the every day lives of a lot of people. Due to restrictions and curfew,
+people from all over the world had to stay home. This also applied to people 
+who were studying or working as well. Business still had work that needed to be done,
+students still had education they needed to finish. However, the world could not simply 
+standby and wait for the pandemic to go over. 
+
+In order to solve this problem,
+various digital solutions have been used to prevent the world economy from receding.
+Technology in form of video communication platforms such as Zoom and Microsoft Teams,
+have allowed students to be able to stll receive teaching/lectures and study material online. 
+Workers, who's work often consists of meetings and communication with other people,
+have also been able to continue doing there work from home through video communication platforms. 
+Restaurants and other businesses in the food industry have been able keep selling food, 
+even with restrictions prohibiting customers to enjoy their food at the restaurant, 
+they have shifted their sales to take away, where customers for example can order
+food from their website or through an app such as Wolt. \newline
+
+
+What do all these digital solutions have in common? They have all been created by 
+some people programming them. Technology can help humans ease or solve these problems, 
+but only if the machine or device have been programmed how to solve these problems. 
+Zoom for example, would not have been able to allow people to have virtual meetings, 
+if it had not been programmed to record each users webcam and microphone, and then output
+it to each others devices. Wolt/restaurants could not have been able to manage
+their orders online as effectively, if it was not for every step in the order process
+being programmed - from displaying the restaurant's menu cart in the app, to make customers
+able to order items from the menu and then send this information to Wolt workers that then 
+deliver the orders. \newline
```

Comment:
```suggestion
it to each others devices. 
```
No need to repeat the arguement of why programmers are important. Has already been covered in the previous sentence.